### ISSUE TITLE
Generator:WindTurbine is missing terms related to C3 and C4 since the C++ conversion

### ISSUE THREAD

**Issue by jmarrec** [NREL/EnergyPlus#8490]
Issue overview
--------------

We're wrapping the object in the OpenStudio SDK.
I went to look for the equation in the source code, and it seems I discovered a bug... 

During the refactor to C++ it seems that the terms of the equation corresponding to `C3` and `C4` have been omitted: 

https://github.com/NREL/EnergyPlus/blob/d6cf92a2a7dc8ca3489d39792c4315f6f3bea957/src/EnergyPlus/WindTurbine.cc#L905-L907

### Details
Some additional details for this issue (if relevant):
 - Platform (Operating system, version): all 
 - Version of EnergyPlus (if using an intermediate build, include SHA): 9.4.0+, d6cf92a2a7dc8ca3489d39792c4315f6f3bea957

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by jmarrec** [comment]
Blaming it, it seems it was perhaps done on purpose: https://github.com/NREL/EnergyPlus/commit/39c4f4b442a87415302661c230c6cd2f4d939024#diff-84854260b17a7faf4f2b6412a950cb5835e1dd62145bd7f2d78c2e3f3a00b3f6R855

`Tuned PitchAngle is zero`. Begs the question of why we ask for a `C3` and `C4` coefficient if unused though.

**Comment by RKStrand** [comment]
@jmarrec Yeah, this looks strange.  It's clear someone has done something intentional in this area but was the change that eliminated C3 and C4 terms intentional?  Looking at the documentation, it still seems like the docs are anticipating that the user will enter those coefficients and the model will use them.
@Myoldmopar How do we find out who made the change mentioned and track down what was happening here?

**Comment by Myoldmopar** [comment]
In Fortran, the PitchAngle was [set to zero](https://github.com/NREL/EnergyPlusArchive/blob/master/SourceCode/WindTurbine.f90#L861):

```
REAL(r64), PARAMETER :: PitchAngle   = 0.0d0      ! No pitch control, i.e. maximum rotor speed
```

As such, those terms in that equation would always be zero.  I am guessing the idea was that if we ever asked for variable pitch angle or something then we would have that capability ready to go.

Daeho (@airguider) added this module [in 2009](https://github.com/NREL/EnergyPlusArchive/commit/e34a99f10a1e84d64498e43ae397d76c96223e1a), and it was zero from the beginning there.
